nix/status-go/mobile/build.nix

```diff
@@ -46,12 +46,25 @@ in buildGoPackage rec {
   # TODO: try removing when go is upgraded to 1.22
   GODEBUG = "netdns=cgo+2";
 
-  preBuild = ''
-    echo 'Generate static files'
-    pushd go/src/$goPackagePath
-    make generate SHELL=$SHELL GO111MODULE=on GO_GENERATE_CMD='go generate'
-    popd
-  '';
+  # Sentry for status-go
+  SENTRY_CONTEXT_NAME = "status-mobile";
+  SENTRY_CONTEXT_VERSION = version;
+
+  gitRoot= "go/src/${goPackagePath}";
+
+   preBuild = ''
+     # To fix fatal: not a git repository (or any of the parent directories): .git
+     git init --quiet --initial-branch=main .
```

Comment:
as discussed with Jakub a more kosher way of handling this is by using a fake git implementation that just outputs the version.
An example is the nix derivation in nimbus-eth2 repo : https://github.com/status-im/nimbus-eth2/blob/10bd5f35bdc7f766e6c55d660e938573fbe71a22/nix/default.nix#L35

I also think it makes sense to extract this change out of this branch and into develop separately.